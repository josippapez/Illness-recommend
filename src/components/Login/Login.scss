.login {
  width: 100%;
  height: 100vh;
  display: flex;
  flex-direction: column;
  background: var(--unnamed-color-e2e2e3) 0% 0% no-repeat padding-box;
  opacity: 1;
  justify-content: center;
  align-items: center;
  .login-site {
    position: relative;
    display: flex;
    padding: 40px;
    .text-pop-up-top {
      -webkit-animation: text-pop-up-top 0.5s
        cubic-bezier(0.25, 0.46, 0.45, 0.94) both;
      animation: text-pop-up-top 0.5s cubic-bezier(0.25, 0.46, 0.45, 0.94) both;
    }
    &__text {
      margin-top: calc(30px + 2vh);
      position: absolute;
      text-align: start;
    }
    .site-logo {
      width: 100%;
      height: 100%;
      -webkit-user-select: none; /* Safari */
      -ms-user-select: none; /* IE 10 and IE 11 */
      user-select: none; /* Standard syntax */
    }
    .site-title {
      color: var(--brand-blue);
      font: normal normal normal 28px/36px "Lato, Bold";
      letter-spacing: 0px;
      opacity: 1;
      font-size: 30px;
    }
    .site-text-under-title {
      margin-top: 10px;
      color: var(--brand-blue);
      font: normal normal normal 16px/24px "Lato, Regular";
      letter-spacing: 0px;
      font-size: 20px;
    }

    @media screen and (max-width: 1210px) {
      flex-direction: column;
      &__text {
        position: fixed;
        top: 0;
      }
      .site-logo {
        height: 50%;
      }
      .login__button-container {
        margin-left: 0;
        margin-top: 40px;
        height: 100%;
      }
      @media screen and (max-width: 425px) {
        flex-direction: column;
        place-items: center;
        .site-logo {
          height: 30%;
          position: relative;
          top: 30px;
        }
        .login__button-container {
          margin-top: 40px;
        }
      }
    }
  }

  &__button-container {
    position: relative;
    width: 100%;
    display: flex;
    justify-content: center;
    align-items: center;
    margin-left: 30px;
    &__register-btn,
    &__login-btn {
      width: calc(160px + 1vw);
      border: none;
      height: calc(193px + 3vh);
      background: var(--white) 0% 0% no-repeat padding-box;
      border-radius: 6px;
      color: var(--brand-blue);
    }
    &__register-btn {
      margin-right: 50px;
      background-position: center 35px;
      background-image: url("../../styles/assets/images/officeBlue.svg");
      background-size: 66px;
      background-repeat: no-repeat;
      cursor: pointer;
      &:hover {
        background: var(--brand-blue) 0% 0% no-repeat padding-box;
        color: var(--unnamed-color-e2e2e3);
        background-image: url("../../styles/assets/images/officeWhite.svg");
        background-size: 66px;
        background-position: center 35px;
        background-repeat: no-repeat;
      }
      &__text {
        position: relative;
        top: 40px;
        text-align: center;
        font: normal normal normal 20px "Lato, Bold";
        letter-spacing: 0px;
        font-size: 25px;
      }
    }
    &__login-btn {
      background-position: center 35px;
      background-image: url("../../styles/assets/images/agendaBlue.svg");
      background-size: 66px;
      background-repeat: no-repeat;
      cursor: pointer;
      &:hover {
        background: var(--brand-blue) 0% 0% no-repeat padding-box;
        color: var(--unnamed-color-e2e2e3);
        background-image: url("../../styles/assets/images/agendaWhite.svg");
        background-size: 66px;
        background-position: center 35px;
        background-repeat: no-repeat;
      }
      &__text {
        position: relative;
        top: 40px;
        text-align: center;
        font: normal normal normal 20px "Lato, Bold";
        letter-spacing: 0px;
        font-size: 25px;
      }
    }
    @media screen and (max-width: 425px) {
      flex-direction: column;
      place-items: center;
      &__register-btn {
        margin-right: 0;
        margin-bottom: 30px;
      }
    }
  }
  &__message {
    width: 100%;
    position: relative;
    top: 40px;
    text-align: center;
    margin-top: 10%;
    color: var(--alert-text);
    font: normal normal bold 20px Helvetica;
  }

  .login-form {
    -webkit-animation: slide-login-bottom 0.2s ease-in-out both;
    animation: slide-login-bottom 0.2s ease-in-out both;
    width: 100%;
    height: 100%;
    position: absolute;
    background: var(--white) 0% 0% no-repeat padding-box;
    &.isClosed {
      -webkit-animation: slide-login-top 0.2s ease-in-out both;
      animation: slide-login-top 0.2s ease-in-out both;
    }
    &__form {
      width: 100%;
      height: 100%;
      .content-container {
        display: flex;
        flex-direction: column;
        justify-content: center;
        align-items: center;
        height: 100%;
        width: 390px;
        margin: auto;
        @media screen and (max-width: 410px) {
          padding: 0px 10px;
          width: 100vw;
        }
        .header {
          position: absolute;
          top: 15%;
          &__back {
            position: absolute;
            top: 15%;
            left: -15%;
            cursor: pointer;
            background-image: url("../../styles/assets/images/ArrowLeft.svg");
            background-repeat: no-repeat;
            height: 20px;
            width: 20px;
            &:hover {
              opacity: 0.7;
            }
            @media screen and (max-width: 410px) {
              left: 0px;
            }
          }
          .sign-in-image {
            background-image: url("../../styles/assets/images/identity.png");
            background-repeat: no-repeat;
            height: 64px;
            width: 64px;
            margin: auto;
          }
          .form-title {
            position: relative;
            top: 20px;
            user-select: none;
            -ms-user-select: none;
            -webkit-user-select: none;
            -moz-user-select: none;
            -webkit-touch-callout: none;
            &__main {
              color: var(--main-text);
              text-align: center;
              font: normal normal bold 28px/36px "Lato, Bold";
              letter-spacing: 0px;
            }
            &__small-text {
              position: relative;
              top: 15px;
              color: var(--light);
              text-align: center;
              font: normal normal normal 16px "Lato, Regular";
              letter-spacing: 0px;
            }
          }
        }
        .body {
          width: 100%;
        }
        &__email,
        &__name {
          z-index: 2;
          padding: 8px 10px;
          font-size: 16px;
          width: 100%;
          height: 38px;
          background-color: var(--white);
          border: 1px solid #dededf;
          border-radius: 4px;
        }
        &__password {
          z-index: 2;
          padding: 8px 40px 8px 10px;
          font-size: 16px;
          width: 100%;
          height: 38px;
          background-color: var(--white);
          border: 1px solid #dededf;
          border-radius: 4px;
        }
        .password-input {
          display: flex;
          flex-direction: column;
          position: relative;
          .password-icon {
            z-index: 2;
            width: 16px;
            height: 16px;
            right: 15px;
            position: absolute;
            &.hide-password {
              top: 12px;
              background-image: url("../../styles/assets/images/HidePassword.svg");
              background-repeat: no-repeat;
            }
            &.show-password {
              top: 14px;
              background-image: url("../../styles/assets/images/ShowPassword.svg");
              background-repeat: no-repeat;
            }
          }
        }
        &__btn {
          position: relative;
          top: 40px;
          height: 38px;
          background: var(--hyperlink) 0% 0% no-repeat padding-box;
          border-radius: 4px;
          border: none;
          color: var(--white);
          text-align: center;
          font: normal normal bold 14px/17px "Lato, Bold";
          letter-spacing: 0px;
          cursor: pointer;
          width: 100%;
          &:hover {
            background: var(--brand-blue) 0% 0% no-repeat padding-box;
          }
        }
      }
    }
  }
}

@-webkit-keyframes slide-login-bottom {
  0% {
    -webkit-transform: translateY(-100vh);
    transform: translateY(-100vh);
  }
  100% {
    -webkit-transform: translateY(0px);
    transform: translateY(0px);
  }
}
@keyframes slide-login-bottom {
  0% {
    -webkit-transform: translateY(-100vh);
    transform: translateY(-100vh);
  }
  100% {
    -webkit-transform: translateY(0px);
    transform: translateY(0px);
  }
}

@-webkit-keyframes slide-login-top {
  0% {
    -webkit-transform: translateY(0px);
    transform: translateY(0px);
  }
  100% {
    -webkit-transform: translateY(-100vh);
    transform: translateY(-100vh);
  }
}
@keyframes slide-login-top {
  0% {
    -webkit-transform: translateY(0px);
    transform: translateY(0px);
  }
  100% {
    -webkit-transform: translateY(-100vh);
    transform: translateY(-100vh);
  }
}

@-webkit-keyframes text-pop-up-top {
  0% {
    -webkit-transform: translateY(50px);
    transform: translateY(50px);
    -webkit-transform-origin: 50% 50%;
    transform-origin: 50% 50%;
    text-shadow: none;
  }
  100% {
    -webkit-transform: translateY(0px);
    transform: translateY(0px);
    -webkit-transform-origin: 50% 50%;
    transform-origin: 50% 50%;
  }
}
@keyframes text-pop-up-top {
  0% {
    -webkit-transform: translateY(50px);
    transform: translateY(50px);
    -webkit-transform-origin: 50% 50%;
    transform-origin: 50% 50%;
    text-shadow: none;
  }
  100% {
    -webkit-transform: translateY(0px);
    transform: translateY(0px);
    -webkit-transform-origin: 50% 50%;
    transform-origin: 50% 50%;
  }
}
