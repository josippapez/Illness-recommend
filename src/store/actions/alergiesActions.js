import axios from 'axios';
import { ALERGIES_LIST_FETCHED, SET_ERROR_ALERGIES_LIST } from '../types';

export const getAllAlergies = () => {
  return (dispatch, getState) => {
    axios({
      method: 'GET',
      url: `${process.env.REACT_APP_API_URL}/alergies`,
      withCredentials: true,
    })
      .then(response => {
        dispatch(setAlergiesList(response.data));
      })
      .catch(error =>
        dispatch({
          type: SET_ERROR_ALERGIES_LIST,
          payload: {
            error: error.message,
            status: error.status,
          },
        })
      );
  };
};

export const setAlergiesList = data => ({
  type: ALERGIES_LIST_FETCHED,
  payload: data,
});
