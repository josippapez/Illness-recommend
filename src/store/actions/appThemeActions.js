import { SET_THEME } from '../types';

export const setDarkTheme = data => ({
  type: SET_THEME,
  payload: data,
});
