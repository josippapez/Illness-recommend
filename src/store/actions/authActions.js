import axios from 'axios';
import { SET_ERROR_USER, USER_LOGGED_IN, USER_LOGGED_OUT } from '../types';

export const registerUser = (name, email, password) => {
  return (dispatch, getState) => {
    axios({
      method: 'POST',
      url: `${process.env.REACT_APP_API_URL}/authentication/register`,
      data: {
        name,
        email,
        password,
      },
    })
      .then(response => {
        dispatch(loginUser(email, password));
      })
      .catch(error =>
        dispatch(
          loginError({
            message: error.response.data.message,
            status: error.response.status,
          })
        )
      );
  };
};

export const loginUser = (email, password) => {
  return (dispatch, getState) => {
    axios({
      method: 'POST',
      url: `${process.env.REACT_APP_API_URL}/authentication/log-in`,
      withCredentials: true,
      data: {
        email,
        password,
      },
    })
      .then(response => {
        dispatch(userLoggedIn(response.data));
      })
      .catch(error =>
        dispatch(
          loginError({
            message: error.response.data.message,
            status: error.response.status,
          })
        )
      );
  };
};

export const userLoggedIn = user => ({
  type: USER_LOGGED_IN,
  payload: {
    id: user.id,
    email: user.email,
    name: user.name,
    role: user.role
  },
});

export const userLoggedOut = () => ({
  type: USER_LOGGED_OUT,
});

export const loginError = error => ({
  type: SET_ERROR_USER,
  payload: {
    error: error.message,
    status: error.status,
  },
});
