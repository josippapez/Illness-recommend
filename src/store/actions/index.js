export * from './authActions';
export * from './usersActions';
export * from './alergiesActions';
export * from './symptomsActions';
export * from './medicationActions';
export * from './patientActions';
