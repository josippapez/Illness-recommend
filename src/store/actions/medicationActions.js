import axios from 'axios';
import Cookies from 'js-cookie';
import {
  MEDICATION_INFO_FETCHED,
  MEDICATION_INFO_FETCHED_ERROR,
  MEDICATION_LIST_FETCHED,
  SET_ERROR_MEDICATION_LIST,
} from '../types';

export const getAllMedications = () => {
  return (dispatch, getState) => {
    axios({
      method: 'GET',
      url: `${process.env.REACT_APP_API_URL}/medications/`,
      headers: {
        Authorization: `Bearer ${Cookies.get('Accesstoken')}`,
      },
    })
      .then(response => {
        dispatch(setMedicationsList(response.data));
      })
      .catch(error =>
        dispatch({
          type: SET_ERROR_MEDICATION_LIST,
          payload: {
            message: error.message,
            error: error.message,
            status: error.status,
          },
        })
      );
  };
};

export const searchMedicationsByText = query => {
  return (dispatch, getState) => {
    axios({
      method: 'GET',
      url: `${process.env.REACT_APP_API_URL}/medications/search?search=${query}`,
      headers: {
        Authorization: `Bearer ${Cookies.get('Accesstoken')}`,
      },
    })
      .then(response => {
        dispatch(setMedicationsList(response.data));
      })
      .catch(error =>
        dispatch({
          type: SET_ERROR_MEDICATION_LIST,
          payload: {
            message: error.message,
            error: error.message,
            status: error.status,
          },
        })
      );
  };
};

export const getMedicationsBySymptomsAndAlergies = symptoms => {
  return (dispatch, getState) => {
    axios({
      method: 'POST',
      url: `${process.env.REACT_APP_API_URL}/medications/find-by-symptoms`,
      headers: {
        Authorization: `Bearer ${Cookies.get('Accesstoken')}`,
      },
      data: symptoms,
    })
      .then(response => {
        dispatch(
          setErrorMedicationInfo({
            message: response.data.successMessage,
            error: null,
            status: response.status,
          })
        );
        dispatch(
          setErrorMedicationInfo({ message: null, error: null, status: null })
        );
        dispatch(setMedicationsList(response.data));
      })
      .catch(error =>
        dispatch({
          type: SET_ERROR_MEDICATION_LIST,
          payload: {
            message: error.message,
            error: error.message,
            status: error.status,
          },
        })
      );
  };
};

export const createMedication = medication => {
  return (dispatch, getState) => {
    axios({
      method: 'POST',
      url: `${process.env.REACT_APP_API_URL}/medications/`,
      headers: {
        Authorization: `Bearer ${Cookies.get('Accesstoken')}`,
      },
      data: medication,
    })
      .then(response => {
        dispatch(
          setErrorMedicationInfo({
            message: response.data.successMessage,
            error: null,
            status: response.status,
          })
        );
        dispatch(
          setErrorMedicationInfo({ message: null, error: null, status: null })
        );
        dispatch(getAllMedications());
      })
      .catch(error => {
        dispatch(
          setErrorMedicationInfo({
            error: error.response.data,
            status: error.status,
          })
        );
      });
  };
};

export const updateMedication = medication => {
  return (dispatch, getState) => {
    axios({
      method: 'PATCH',
      url: `${process.env.REACT_APP_API_URL}/medications/`,
      headers: {
        Authorization: `Bearer ${Cookies.get('Accesstoken')}`,
      },
      data: medication,
    })
      .then(response => {
        dispatch(
          setErrorMedicationInfo({
            message: response.data.successMessage,
            error: null,
            status: response.status,
          })
        );
        dispatch(
          setErrorMedicationInfo({ message: null, error: null, status: null })
        );
        dispatch(getAllMedications());
      })
      .catch(error =>
        dispatch(
          setErrorMedicationInfo({
            error: error.response.data,
            status: error.status,
          })
        )
      );
  };
};

export const removeMedicationById = id => {
  return (dispatch, getState) => {
    axios({
      method: 'DELETE',
      url: `${process.env.REACT_APP_API_URL}/medications/delete`,
      headers: {
        Authorization: `Bearer ${Cookies.get('Accesstoken')}`,
      },
      data: { id },
    })
      .then(response => {
        dispatch(
          setErrorMedicationInfo({
            message: response.data.successMessage,
            error: null,
            status: response.status,
          })
        );
        dispatch(
          setErrorMedicationInfo({ message: null, error: null, status: null })
        );
        dispatch(getAllMedications());
      })
      .catch(error =>
        dispatch(
          setErrorMedicationInfo({
            error: error.response.data,
            status: error.status,
          })
        )
      );
  };
};

export const setMedicationsList = data => ({
  type: MEDICATION_LIST_FETCHED,
  payload: data,
});

export const medicationInfoFetched = data => ({
  type: MEDICATION_INFO_FETCHED,
  payload: data,
});

export const setErrorMedicationInfo = data => ({
  type: MEDICATION_INFO_FETCHED_ERROR,
  payload: data,
});
