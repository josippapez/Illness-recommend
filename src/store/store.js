import { createStore, applyMiddleware } from 'redux';
import { persistStore, persistReducer } from 'redux-persist';
import storage from 'redux-persist/lib/storage';
import logger from 'redux-logger';
import thunk from 'redux-thunk';

import { reducers } from './reducers';

const middleware = [];

const persistConfig = {
  key: 'root',
  storage,
};
const persistedReducer = persistReducer(persistConfig, reducers);

middleware.push(thunk);
if (process.env.NODE_ENV === 'development') middleware.push(logger);

const store = createStore(persistedReducer, applyMiddleware(...middleware));

const persistor = persistStore(store);

export { persistor, store };
