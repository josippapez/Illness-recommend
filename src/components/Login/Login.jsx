import React, { useEffect, useState } from 'react';
import { useDispatch, useSelector } from 'react-redux';
import { loginUser, registerUser } from '../../store/actions';

function Login() {
  const user = useSelector(state => state.user);
  const dispatch = useDispatch();

  const [name, setName] = useState(null);
  const [email, setEmail] = useState(null);
  const [password, setPassword] = useState(null);

  useEffect(() => {
    if (user && user.id) {
      window.location.replace('/');
    }
  }, [user]);

  return (
    <div className="Login">
      <header className="Login-header">
        <div>registracija</div>
        <input
          type="text"
          name="name"
          value={name ? name : ''}
          onChange={e => {
            setName(e.target.value);
          }}
        />

        <input
          type="email"
          name="email"
          value={email ? email : ''}
          onChange={e => {
            setEmail(e.target.value);
          }}
        />

        <input
          type="password"
          name="password"
          value={password ? password : ''}
          onChange={e => {
            setPassword(e.target.value);
          }}
        />

        <button
          onClick={() => {
            dispatch(registerUser(name, email, password));
          }}
        >
          Pošalji
        </button>

        <div>Prijava</div>

        <input
          type="email"
          name="email"
          value={email ? email : ''}
          onChange={e => {
            setEmail(e.target.value);
          }}
        />

        <input
          type="password"
          name="password"
          value={password ? password : ''}
          onChange={e => {
            setPassword(e.target.value);
          }}
        />

        <button
          onClick={() => {
            dispatch(loginUser(email, password));
          }}
        >
          Pošalji
        </button>
      </header>
    </div>
  );
}

export default Login;
