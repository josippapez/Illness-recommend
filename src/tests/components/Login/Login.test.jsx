/* eslint-disable no-undef */
import React from 'react';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
import { mount } from 'enzyme';

import { loginError, loginUser, registerUser } from '../../../store/actions';
import Login from '../../../components/Login/Login';

jest.mock('../../../store/actions');

const initialState = {
  user: {
    userInfo: {
      data: null,
      error: null,
      status: null,
    },
    role: null,
    isAuthenticated: false,
    error: [
      {
        message: 'Polje je obavezno',
        field: 'email',
      },
      {
        message: 'Polje je obavezno',
        field: 'password',
      },
      {
        message: 'Polje je obavezno',
        field: 'name',
      },
      {
        message: 'Vrijednost nije pravilnog formata',
        field: 'name',
      },
    ],
    status: 400,
  },
  errorMessage: { message: 'Test' },
};

const mockStore = configureStore();
const store = mockStore(initialState);

describe('Login', () => {
  let component;
  delete window.location;

  beforeEach(() => {
    window.location = { replace: jest.fn() };
    Object.defineProperty(window.location, 'replace', {
      configurable: true,
    });
    window.location.replace = jest.fn();

    registerUser.mockImplementation(() => ({ type: 'test' }));
    loginUser.mockImplementation(() => ({ type: 'test' }));

    component = mount(
      <Provider store={store}>
        <Login />
      </Provider>
    );
  });

  it('renders correctly', () => {
    expect(component.html()).toMatchSnapshot();
  });

  it('it should show an error message if there is error', () => {
    window.location.replace('/login?message=error');
    jest.spyOn(URLSearchParams.prototype, 'get').mockImplementation(key => key);
    loginError.mockImplementation(message => ({
      type: 'TEST',
      payload: { message },
    }));
    const newComponent = mount(
      <Provider store={store}>
        <Login />
      </Provider>
    );
    expect(newComponent.find('.login__message')).toBeTruthy();
  });

  describe('form', () => {
    it('it should show a form when clicked on non-student login button', () => {
      const registerBtn = component.find(
        '.login__button-container__register-btn'
      );
      const form = component.find('.login__form');
      registerBtn.simulate('click');
      expect(form).toBeTruthy();
    });

    it('it should remove a form when clicked on return button', () => {
      loginError.mockImplementation(() => ({
        type: 'TEST',
      }));
      component
        .find('.login__button-container__register-btn')
        .simulate('click');
      component.find('.header__back').simulate('click');
      expect(loginError).toBeCalled();
    });

    it('email and password content changes on input', () => {
      component
        .find('.login__button-container__register-btn')
        .simulate('click');

      component.find('.content-container__name').simulate('change', {
        target: { value: 'test' },
      });
      component.find('.content-container__email').simulate('change', {
        target: { value: 'test' },
      });
      component.find('.content-container__password').simulate('change', {
        target: { value: 'test' },
      });
      component.find('.password-icon').simulate('click');
      expect(component.find('.content-container__name').props().value).toBe(
        'test'
      );
      expect(component.find('.content-container__email').props().value).toBe(
        'test'
      );
      expect(component.find('.content-container__password').props().value).toBe(
        'test'
      );
    });

    it('it should dispatch login user when login form button is clicked', () => {
      loginUser.mockImplementation((email, password) => ({
        type: 'TEST',
        payload: { email, password },
      }));
      loginError.mockImplementation(error => ({
        type: 'TEST',
        payload: error,
      }));
      component.find('.login__button-container__login-btn').simulate('click');
      component.find('.content-container__email').simulate('change', {
        target: { value: 'test' },
      });
      component.find('.content-container__password').simulate('change', {
        target: { value: 'test' },
      });
      component.find('form').simulate('submit');
      expect(loginUser).toBeCalled();
      component.unmount();
    });
  });
});
