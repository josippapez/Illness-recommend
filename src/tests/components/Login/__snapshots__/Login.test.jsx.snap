// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login renders correctly 1`] = `"<div class=\\"login\\"><div class=\\"login-site\\"><div class=\\"login-site__text text-pop-up-top\\"><div class=\\"site-title\\">Preporuke lijekova</div><div class=\\"site-text-under-title\\">Odaberite jednu od opcija za nastavak.</div></div><img src=\\"undraw_Mobile_login_re_9ntv.svg\\" alt=\\"background\\" class=\\"site-logo\\"><div class=\\"login__button-container\\"><button class=\\"login__button-container__register-btn\\" type=\\"button\\"><div class=\\"login__button-container__register-btn__text\\">Registracija</div></button><button class=\\"login__button-container__login-btn\\" type=\\"button\\"><div class=\\"login__button-container__login-btn__text\\">Prijava</div></button></div></div><form class=\\"\\"></form></div>"`;
